prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.html

```diff
@@ -37,7 +37,8 @@
               {{ setting }}
             </ng-container>
             <ng-container appPageSubheaderSummary>
-              {{ subheaderContent }}
+              {{ subheaderContent }} If you work in more than one clinic you can include the email addresses for all the
+              PharmaNet administrators separated by a comma.
```

Comment:
You should remove `<app-enrolment-progress-indicator *ngIf="showProgressBar"></app-enrolment-progress-indicator>` from this file as well. otherwise looks great!

Reply:
Don.e